gnd/src/main/java/com/google/android/gnd/persistence/geojson/GeoJsonParser.java

```diff
@@ -96,8 +96,7 @@ private String readJsonFile(File file) {
 
     for (int i = 0; i < arr.length(); i++) {
       try {
-        JSONObject o = arr.getJSONObject(i);
-        result[i] = o;
+        result[i] = arr.getJSONObject(i);
       } catch (JSONException e) {
         Log.e(TAG, "couldn't parse json", e);
```

Comment:
This error message is slightly misleading "Ignoring Error in JSON" might be more accurate.

Reply:
Good call, updated to "Ignoring error in JSON array"